WORKSPACE

```diff
@@ -97,9 +97,16 @@ load("@io_grpc_grpc_java//:repositories.bzl", "grpc_java_repositories")
 
 grpc_java_repositories()
 
-load("@envoy_api//bazel:repositories.bzl", "api_dependencies")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```

Comment:
nit: would it make sense to add a quick explainer on how grpc needs this and maybe reference https://github.com/grpc/grpc-java/pull/12243?

Reply:
I think the WORKSPACE file in googleapis needs this change as well, can we follow up on that once this change is released?